source/sass/glyphs.scss

```diff
@@ -18,6 +18,19 @@
   &::before {
     @include glyph($id);
   }
+
+  &.gray {
+    &::before {
+      filter: opacity(40%);
```

Comment:
this is a cheat but I don't know what else we can do other than adding grey version of SVGs in